eleventy.config.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { EleventyRenderPlugin } = require('@11ty/eleventy');
+const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
+const yaml = require('js-yaml');
+
+const codeExample = require('./source/helpers/codeExample.ts').default;
+const compatibility = require('./source/helpers/compatibility.ts');
+const components = require('./source/helpers/components.ts');
+const dates = require('./source/helpers/dates.ts');
+const { liquidEngine, markdownEngine } = require('./source/helpers/engines.ts');
+const page = require('./source/helpers/page.ts');
+const type = require('./source/helpers/type.ts');
```

Comment:
I'm happy for feedback on how I organized and named these.

Reply:
The other thing that we can do to clean it up a bit is have each individual file do an internal `module.exports = (eleventyConfig) => { ... }` with the added filters/codes, and then import them here as plugins. Don't know if that would feel a bit more organized?

Reply:
@mirisuzanne Adjusted in https://github.com/oddbird/sass-site/commit/ad8a53c4a57a9df7efd8c0a06f1be05af34e86d5